includes/net-standard-1.0.md

```diff
@@ -0,0 +1,18 @@
+.NET Standard 1.0 has 7,949 of the 37,118 available APIs.
+
+| .NET implementation         | Version support                                              |
+|-----------------------------|--------------------------------------------------------------|
```

Comment:
Should we remove the non-LTS versions that are out of support here? Or is this meant to be a historically accurate statement?

Reply:
This is meant to highlight all possible implementations of .NET Standard versions. It's a map of what .NET Implementation aligns with a given .NET Standard version.